ledger/src/ui/test/attestation/test_attestation.c

```diff
@@ -216,19 +217,23 @@ void test_get_attestation_get_msg() {
 
     // CLA + INS_ATTESTATION + ATT_OP_GET_MSG + PAGE_NUM (0)
     set_apdu("\x80\x50\x02\x00");
-
-    int ret = get_attestation(4, &G_att_ctx);
-    assert((ATT_MESSAGE_SIZE + 4) == get_attestation(4, &G_att_ctx));
+    assert((APDU_TOTAL_DATA_SIZE_OUT + 3) == get_attestation(4, &G_att_ctx));
     ASSERT_APDU(
```

Comment:
This seems odd. If there are more pages, then the response at the fourth byte should have an `\x01` indicating that. Can you double check please? (see `attestation.c:274`). Maybe an actual bug?